packages/css-library/dist/stylesheets/_functions.scss

```diff
@@ -0,0 +1,32 @@
+@import "./formation-overrides/variables";
+@import "./override-function";
+@charset "UTF-8";
+
+
+// Spacing Units
+@function units($unit) {
+  //$name: nth($flex-names, $i);
+
+  @if not map-has-key($units, $unit) {
+    @error '`#{$unit}` is not a valid spacing unit token. '
+      + 'Valid spacing unit tokens: '
+      + '#{map-keys($units)}';
+  }
+
+  $val: map-get($units, $unit) ;
+  $remVal: $val + rem;
+  @return scale-rem($remVal);
+}
+
+@function units-px($unit) {
+  //$name: nth($flex-names, $i);
+
+  @if not map-has-key($units, $unit) {
+    @error '`#{$unit}` is not a valid spacing unit token. '
+      + 'Valid spacing unit tokens: '
+      + '#{map-keys($units)}';
+  }
+
+  $val: map-get($units, $unit) ;
+  @return ($val * 10) + px;
+}
```

Comment:
These are direct ports from formation - https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/base/_b-functions.scss#L169-L194